cruise-control/src/integrationTest/java/com/linkedin/kafka/cruisecontrol/ReplicaCapcaityViolationIntegrationTest.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2020 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See
+ * License in the project root for license information.
```

Comment:
Nit#1: As a convention, CC repo uses the year the file has been created in copyright: `2020` -> `2021`.
Nit#2: Can we avoid adding a new line after `See`?